Content.Server/_DV/CosmicCult/Components/CosmicAstralBodyComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Content.Server._DV.CosmicCult.Abilities;
+
+namespace Content.Server._DV.CosmicCult.Components;
+
+[RegisterComponent, Access(typeof(CosmicReturnSystem))]
+[AutoGenerateComponentPause]
+public sealed partial class CosmicAstralBodyComponent : Component
```

Comment:
`Class 'CosmicAstralBodyComponent' has [AutoGenerateComponentPause] but has no fields or properties with [AutoPausedField]`
```suggestion
public sealed partial class CosmicAstralBodyComponent : Component
```